Dockerfile

```diff
@@ -36,6 +36,7 @@ RUN set -x \
 
 WORKDIR $AIRFLOW_HOME
 RUN mkdir -p $AIRFLOW_HOME
+RUN airflow initdb
```

Comment:
Do not think we need `airflow initdb` here. L41 had `ENTRYPOINT`
https://github.com/apache/airflow/blob/067a1e3f4aa153195674950fee72bdd23652e61d/scripts/docker/entrypoint.sh#L24-L26
which exec `airflow` command including `airflow initdb`
```suggestion
```